src/main/java/com/faforever/client/fa/ForgedAllianceService.java

```diff
@@ -97,8 +104,15 @@ public Process startReplay(Path path, @Nullable Integer replayId) throws IOExcep
   }
 
 
-  public Process startReplay(URI replayUri, Integer replayId) throws IOException {
-    List<String> launchCommand = defaultLaunchCommand().replayUri(replayUri)
+  public Process startReplay(URI replayUri, Integer replayId, boolean useReplayFolder) throws IOException {
```

Comment:
We already have ForgedAlliancePrefs injected. just read directly from that.

Reply:
We could do the same in `GameBinariesUpdateTaskImpl`. For the rest we would need to inject ForgedAlliancePrefs first. Should I refactor these as well?

Reply:
Well for the game binaries updater that function is actually used for both the regular game and the replay game. And the prefs value is not sufficient to use since it can be true when we are trying to update the regular game files.

Reply:
Right, good catch